### PR TITLE
r2.0 cherry-pick request: [Intel MKL] Upgrading curl to 7.65.3 to fix CVE-2019-5443

### PR DESCRIPTION
**Pull request by penpornk**
Description from the original PR https://github.com/tensorflow/tensorflow/pull/31675:

# [CVE-2019-5443](https://nvd.nist.gov/vuln/detail/CVE-2019-5443)

**NVD:** 2019/07/02 - CVSS v2.0 Base Score: 4.6 - CVSS v3.0 Base Score: 7.8

`A non-privileged user or program can put code and a config file in a known non-privileged path (under C:/usr/local/) that will make curl <= 7.65.1 automatically run the code (as an openssl "engine") on invocation. If that curl is invoked by a privileged user it can do anything it wants.`

## References to Advisories, Solutions, and Tools
Source | Link | Type
---- | ---- | ----
MLIST | [www.openwall.com](http://www.openwall.com/lists/oss-security/2019/06/24/1) | Mailing List, Patch, Third Party Advisory
BID | [www.securityfocus.com](http://www.securityfocus.com/bid/108881) | Third Party Advisory, VDB Entry
MISC | [curl.haxx.se](https://curl.haxx.se/docs/CVE-2019-5443.html) | Patch, Vendor Advisory

### CODE REVIEW

**Review by gunan — DISMISSED** [review]

File: tensorflow/workspace.bzl [hunk 1]
```diff
@@ -497,12 +497,12 @@ def tf_repositories(path_prefix = "", tf_repo_name = ""):
...
         build_file = clean_dep("//third_party:curl.BUILD"),
-        sha256 = "e9c37986337743f37fd14fe8737f246e97aec94b39d1b71e8a5973f72a9fc4f5",
-        strip_prefix = "curl-7.60.0",
+        sha256 = "4376ac72b95572fb6c4fbffefb97c7ea0dd083e1974c0e44cd7e49396f454839",
+        strip_prefix = "curl-7.65.3",
         system_build_file = clean_dep("//third_party/systemlibs:curl.BUILD"),
         urls = [
-            "https://storage.googleapis.com/mirror.tensorflow.org/curl.haxx.se/download/curl-7.60.0.tar.gz",
-            "https://curl.haxx.se/download/curl-7.60.0.tar.gz",
+            "http://mirror.tensorflow.org/curl.haxx.se/download/curl-7.65.3.tar.gz",
```
Comment (gunan) [inline tensorflow/workspace.bzl]: Could you update this URL to 
https://storage.googleapis.com/mirror.tensorflow.org/curl.haxx.se/download/curl-7.65.3.tar.gz

Comment (penpornk) [reply tensorflow/workspace.bzl]: Done. Thank you! I'll fix this in the master branch too.

**Comment by penpornk** [comment]
Manually setting the CLA to yes again since the other commit that I didn't author is already merged in the master branch.

**Comment by claynerobison** [comment]
@googlebot I consent.

**Review by gunan — APPROVED** [review]

**Pull request merged**